### ISSUE TITLE
Community-driven auto moderation

### ISSUE THREAD

**Issue by tk3369** [Humans-of-Julia/HoJBot.jl#34]
# Background 
A primary job of moderators is to ensure a comfortable environment for community members to interact with each other. For a diverse community with people coming from different culture, background, age group, etc., it is even more important to take this seriously.

The primary problems for manual moderation are:
1. It is sometimes difficult to decide whether something is offensive
2. The mods takes a lot of effort to discuss whether the content is appropriate or should be removed
3. It takes time to reach a consensus/decision before action can be taken

# Proposed approach
I propose that we automate this moderation work with the HoJBot. It could work as follows:

1. Member A posts something inappropriate
2. Member B felt uncomfortable and reacted with an `X` emoji.
3. Member C also felt uncomfortable and reacted with an `X` emoji.
4. etc.
5. When it reaches a threshold (e.g. 3 `X`'s) then the bot removes the post and notifies the mods & member A about the removal.

So basically, we let the community decides whether something is bad.

_Originally posted by @tk3369 in https://github.com/Humans-of-Julia/HoJBot.jl/discussions/2_

**Comment by tk3369** [comment]
This feature is now live!

There are actually few more things that I want to do. I am going to submit those issues separately.
